src/Artifacts.jl

```diff
@@ -221,6 +221,132 @@ function topmodel_data_path(; context = nothing)
     )
     path = joinpath(get_data_folder(topmodel_dataset), "means_2.5_new.nc")
     return path
+
+end
+
+"""
+    lehmann_assouline_or2008_evaporation_data(; context=nothing)
+
+Local path to file containing measured evaporation rate as a function of time
+for bare soil.
+
+Data was originally collected by Lehmann, Peter, Shmuel Assouline, 
+and Dani Or. "Characteristic lengths affecting evaporative drying of 
+porous media." Physical Review E 77.5 (2008): 056309 and presented
+in Figure 8 of that work.
+
+https://doi.org/10.1103/PhysRevE.77.056309
+"""
+function lehmann_assouline_or2008_evaporation_data(; context = nothing)
+    evap_dataset = ArtifactWrapper(
+        @__DIR__,
```

Comment:
Do we mabye need `joinpath(@__DIR__, "..")?

Reply:
yes, I forgot! good catch

Reply:
Good to go once you fix that!